packages/website/pages/terms.js

```diff
@@ -23,75 +23,158 @@ export default function TermsOfService() {
+            Verified Deals
+          </a>{' '}
+          , which Filecoin Storage Providers are incentivized to accept at a lower 
+          price vs. other deals because they have a relatively higher increase in 
+          their chance to win block rewards.
+        </p>
+        <p className="lh-copy">
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is committed to ensuring that all NFTs on Filecoin have a 
+          perpetual storage solution. All Filecoin storage deals facilitated by 
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} are automatically renewed in perpetuity by {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} before 
+          expiration. In late 2022, we plan to build tooling to enable the 
+          non-custodial, perpetual renewal of storage contracts independent of 
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} to secure the perpetual persistence of all NFTs without 
+          dependence on any single entity.
+        </p>
+        <p className="lh-copy">
+          In addition to storing data in Filecoin deals, {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} currently 
+          stores data on a self-managed IPFS Cluster with 6 nodes, with any given 
+          data replicated on 3 of them. These nodes are used to (1) store data 
+          while data is being aggregated into Filecoin deals, and (2) provide 
+          data to the public IPFS network for retrieval. This IPFS Cluster 
+          infrastructure may undergo changes over time, such as migrations or 
+          upgrades. Additional content-addressed copies of the data are also 
+          kept outside this IPFS Cluster.
+        </p>
+        <p className="lh-copy">
+          Data stored via {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is guaranteed to be publicly available via 
+          IPFS, and users are responsible and liable for the data they upload. 
+          All data uploaded is available to anyone who requests it 
+          using the correct CID. Users should not store any private or sensitive 
+          information in an unencrypted form using {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '}. Further, deleting 
+          files from {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} will remove them from the file listing for a user's 
+          account, but nodes on the IPFS network may retain copies of the data indefinitely. 
```

Comment:
...and data aggregated into Filecoin deals will persist for the duration of the deal.